manim/animation/transform.py

```diff
@@ -227,6 +227,30 @@ def interpolate(self, alpha: float) -> None:
 
 
 class ClockwiseTransform(Transform):
+    """Transforms Mobjects while moving them in Clockwise direction.
+
+    Examples
+    --------
+
+    .. manim:: ClockwiseTransform1
+        :quality: medium
+
+        class ClockwiseTransform1(Scene):
+            def construct(self):
+                self.play(ClockwiseTransform(Square(), Circle()))
+
+    .. manim:: ClockwiseTransform2
```

Comment:
Regarding naming of the `ClockwiseTransform` and `CounterclockwiseTransform` examples: I prefer the naming scheme you used for the other classes, i.e., `ClassnameExample` over just `Classname42`.

Reply:
i'll change them

Reply:
Did you take any other examples from YouTube or other sources? Keep in mind that the original authors of the examples must explicitly agree with their use in our documentation, and you should provide attribution. Are all the other examples in this PR (and your other PRs as well) your original work?

Reply:
> Did you take any other examples from YouTube or other sources? Keep in mind that the original authors of the examples must explicitly agree with their use in our documentation, and you should provide attribution. Are all the other examples in this PR (and your other PRs as well) your original work?

no, I only took examples from @behackl 's channel

I'll keep that in mind the next time add examples